relay.go

```diff
@@ -281,9 +281,20 @@ func (r *Relayer) Receive(f *Frame, fType frameType) (sent bool, failureReason s
 	case r.conn.sendCh <- f:
 	default:
 		// Buffer is full, so drop this frame and cancel the call.
-		r.logger.WithFields(
-			LogField{"id", id},
-		).Warn("Dropping call due to slow connection.")
+
+		// Since this is typically due to the send buffer being full, get send buffer
+		// usage + limit and add that to the log.
+		sendBuf, sendBufLimit, sendBufErr := r.conn.sendBufSize()
+		logFields := []LogField{
+			{"id", id},
+			{"destConnSendBufferCurrent", sendBuf},
+			{"destConnSendBufferLimit", sendBufLimit},
+		}
+		if sendBufErr != nil {
+			logFields = append(logFields, LogField{"destConnSendBufferError", sendBufErr.Error()})
+		}
+
+		r.logger.WithFields(logFields...).Warn("Dropping call due to slow connection.")
```

Comment:
I think we should we also log `len(sendCh)`, since that's gonna be configurable and may differ depending on the connection.

Reply:
Good idea, I've added both `len(sendCh)` and `cap(sendCh)` to both the log output as well as the introspection output.